pkg/services/featuremgmt/toggles_gen.go

```diff
@@ -271,6 +271,10 @@ const (
 	// Use Elasticsearch as backend data source
 	FlagElasticsearchBackendMigration = "elasticsearchBackendMigration"
 
+	// FlagDatasourceOnboarding
+	// Enable data source onboarding page
+	FlagDatasourceOnboarding = "datasourceOnboarding"
```

Comment:
@kaydelaney @thanos-karachalios - have you discussed if we want to keep this behind feature flag after merging?